main.py

```diff
@@ -91,11 +97,11 @@ def main(config):
     if config.n_samples > 0 and config.n_samples <= 1e5:
         batch, times = gflownet.sample_batch(n_forward=config.n_samples, train=False)
         x_sampled = batch.get_terminating_states(proxy=True)
-        energies = env.proxy(x_sampled)
+        energies = gflownet.env.proxy(x_sampled)
```

Comment:
This will result in a conflict.

Reply:
Done. It was fixed locally and I just pushed the changes.